### PR TITLE
Fix in readme manager key for mongodb

### PR DESCRIPTION
**Pull request by ne0h12**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ require_once './vendor/autoload.php';
 Each kind of manager has its configurations stored on a key in the settings array
  
 * `ManagerBuilder::DEFAULT_RELATIONAL_MANAGER_KEY` ("entity_manager") for `ORM`
-* `ManagerBuilder::DEFAULT_RELATIONAL_MANAGER_KEY` ("mongodb_document_manager") for `MongoDB ODM`
+* `ManagerBuilder::MONGODB_MANAGER_KEY` ("mongodb_document_manager") for `MongoDB ODM`
```
Comment (juliangut) [inline README.md]: It should be ManagerBuilder::DEFAULT_MONGODB_MANAGER_KEY

File: README.md [hunk 2]
```diff
@@ -49,7 +49,7 @@ Each kind of manager has its configurations stored on a key in the settings arra
         'manager2_name' => <relational_manager_builder_configuration>,
         ...
     ],
-    ManagerBuilder::DEFAULT_RELATIONAL_MANAGER_KEY => [
+    ManagerBuilder::MONGODB_MANAGER_KEY => [
```
Comment (juliangut) [inline README.md]: It should be ManagerBuilder::DEFAULT_MONGODB_MANAGER_KEY

**Comment by juliangut** [comment]
Thanks for the fix, review my comments and it's ready for merge

**Comment by ne0h12** [comment]
Sorry for my inattention, I fixed this.

**Comment by juliangut** [comment]
Thanks

**Pull request merged**